pkg/meta/sql.go

```diff
@@ -434,7 +438,9 @@ func (m *dbMeta) Lookup(ctx Context, parent Ino, name string, inode *Ino, attr *
 			// TODO: in SQL
 			if e := m.resolveCase(ctx, parent, name); e != nil {
 				*inode = e.Inode
-				*attr = *e.Attr
+				if attr != nil && e.Attr != nil {
```

Comment:
if attr is not nil, but e.Attr is, we need fill the attr using GetAttr()

Reply:
OK, that needs another fix. I'll revert this change for now.